sql/src/main/java/io/druid/sql/avatica/DruidStatement.java

```diff
@@ -184,7 +198,7 @@ public DruidStatement execute()
         catch (Throwable t1) {
           t.addSuppressed(t1);
         }
-        throw t;
+        Throwables.propagate(t);
```

Comment:
`throw Throwables.propagate(t)` is better -- it lets the compiler and any analysis tools understand that there's an exception being thrown.